### PR TITLE
migrate from Travis CI to GitHub Actions CI

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8', '11', '13', '14' ]
```
Comment (OlegDokuka) [inline .github/workflows/ci.yml:16]:
```suggestion
        java: [ '8', '11', '13' ]
```

Just FYI. The higher support version by the gradle 6.0.1 is JDK 13. Support for JDK 14 starts from the gradle 6.3+, however and the same time, starting from gradle 6.1+ there is no support for JDK 1.6 which is a baseline for this project

Comment (sullis) [reply .github/workflows/ci.yml]: LGTM

**Comment by viktorklang** [comment]
@reactive-streams/contributors Any thoughts or objections towards switching to GH Actions CI instead of TravisCI?

File: .travis.yml [hunk 2]
```diff
@@ -1,35 +0,0 @@
...
-# Don't let Travis CI execute './gradlew assemble' by default
-# From https://github.com/reactive-streams/reactive-streams-jvm/pull/383
-install:
-# Display Gradle, JVM and other versions
-  - ./gradlew -version
-
-env:
-  global:
-  - TERM=dumb
-  - DEFAULT_TIMEOUT_MILLIS=300
```
Comment (viktorklang) [inline .travis.yml:31]: @sullis What about these TCK ENV vars?

**Review by akarnokd — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (akarnokd) [reply .travis.yml:31]: Not in use anymore?

https://github.com/reactive-streams/reactive-streams-jvm/search?q=DEFAULT_TIMEOUT_MILLIS

**Comment by akarnokd** [comment]
> Any thoughts or objections towards switching to GH Actions CI instead of TravisCI?

Travis CI recently put many of us to their free but severely limited OSS plan (1000 build minutes credit and the need to keep renegotiating every time it runs out). I'd say for the CI build needs of this project, GitHub Actions is quite adequate.

**Comment by sullis** [comment]
ping @akarnokd

**Review by akarnokd — APPROVED** [review]

**Comment by sullis** [comment]
can you merge this? @akarnokd

**Comment by akarnokd** [comment]
I don't have the rights.

**Comment by viktorklang** [comment]
@rkuhn Would you be able to merge this, I'm currently not on a machine
where I have access.

**Comment by rkuhn** [comment]
done

**Comment by rkuhn** [comment]
build failed: https://github.com/reactive-streams/reactive-streams-jvm/runs/2827968925?check_suite_focus=true#step:5:31

Any gradle folks around to fix it? Otherwise I’ll need to revert.

**Pull request merged**